app/Config/core.php

```diff
@@ -237,47 +238,47 @@
  * Set to `true` to apply timestamps when debug > 0. Set to 'force' to always enable
  * timestamping regardless of debug value.
  */
-	//Configure::write('Asset.timestamp', true);
+    //Configure::write('Asset.timestamp', true);
 
-/**
+/*
  * Compress CSS output by removing comments, whitespace, repeating tags, etc.
  * This requires a/var/cache directory to be writable by the web server for caching.
  * and /vendors/csspp/csspp.php
  *
  * To use, prefix the CSS link URL with '/ccss/' instead of '/css/' or use HtmlHelper::css().
  */
-	//Configure::write('Asset.filter.css', 'css.php');
+    //Configure::write('Asset.filter.css', 'css.php');
 
-/**
+/*
  * Plug in your own custom JavaScript compressor by dropping a script in your webroot to handle the
  * output, and setting the config below to the name of the script.
  *
  * To use, prefix your JavaScript link URLs with '/cjs/' instead of '/js/' or use JsHelper::link().
  */
-	//Configure::write('Asset.filter.js', 'custom_javascript_output_filter.php');
+    //Configure::write('Asset.filter.js', 'custom_javascript_output_filter.php');
```

Comment:
__[staging][code_sniffer]__
* Comment closer must be on a new line
* No space found before comment text; expected "// Configure::write('Asset.filter.js', 'custom_javascript_output_filter.php');" but found "//Configure::write('Asset.filter.js', 'custom_javascript_output_filter.php');"
* Inline comments must end in full-stops, exclamation marks, or question marks
* There must be no blank line following an inline comment